pkg/service/v0/errorcode/errorcode.go

```diff
@@ -0,0 +1,75 @@
+package errorcode
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+	msgraph "github.com/yaegashi/msgraph.go/v1.0"
+)
+
+// ErrorCode defines code as used in MS Graph - see https://docs.microsoft.com/en-us/graph/errors?context=graph%2Fapi%2F1.0&view=graph-rest-1.0
+type ErrorCode int
+
+const (
+	// AccessDenied : The caller doesn't have permission to perform the action.
+	AccessDenied ErrorCode = iota
+	// ActivityLimitReached : The app or user has been throttled.
+	ActivityLimitReached
+	// GeneralException : An unspecified error has occurred.
+	GeneralException
+	// InvalidRange : The specified byte range is invalid or unavailable.
+	InvalidRange
+	// InvalidRequest : The request is malformed or incorrect.
+	InvalidRequest
+	// ItemNotFound : The resource could not be found.
+	ItemNotFound
+	// MalwareDetected : Malware was detected in the requested resource.
+	MalwareDetected
+	// NameAlreadyExists : The specified item name already exists.
+	NameAlreadyExists
+	// NotAllowed : The action is not allowed by the system.
+	NotAllowed
+	// NotSupported : The request is not supported by the system.
+	NotSupported
+	// ResourceModified : The resource being updated has changed since the caller last read it, usually an eTag mismatch.
+	ResourceModified
+	// ResyncRequired : The delta token is no longer valid, and the app must reset the sync state.
+	ResyncRequired
+	// ServiceNotAvailable : The service is not available. Try the request again after a delay. There may be a Retry-After header.
+	ServiceNotAvailable
+	// QuotaLimitReached : The user has reached their quota limit.
+	QuotaLimitReached
+	// Unauthenticated : The caller is not authenticated.
+	Unauthenticated
+)
+
+var errorCodes = [...]string{
```

Comment:
extra points for using `[...]` 👍

Reply:
THX - but I only copied this from some where :see_no_evil: